user_guide_src/source/models/model/015.php

```diff
@@ -5,4 +5,4 @@
     'email'    => 'd.vader@theempire.com',
 ];
 
-$userModel->insert($data);
+$userModel->insert($data, bool $returnID = true);
```

Comment:
Please revert. It is syntax error, and `$returnID = true`  is the default value, we don't need it.